src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/ObjectWriter/Dwarf/DwarfInfo.cs

```diff
@@ -0,0 +1,585 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using ILCompiler.DependencyAnalysis;
+using Internal.JitInterface;
+using Internal.TypeSystem;
+using Internal.TypeSystem.TypesDebugInfo;
+using static ILCompiler.ObjectWriter.DwarfNative;
+
+namespace ILCompiler.ObjectWriter
+{
+    internal abstract class DwarfInfo
+    {
+        public abstract void Dump(DwarfInfoWriter writer);
+    }
+
+    internal sealed class DwarfPrimitiveTypeInfo : DwarfInfo
+    {
+        private TypeFlags _typeFlags;
+        private int _targetPointerSize;
```

Comment:
readonly

Reply:
I already had the change locally. Just forgot to save and commit it :)